pages/blog/status-update-37-20.md

```diff
@@ -63,11 +63,11 @@ Please have a look at [this](https://github.com/asyncapi/asyncapi/issues/432) is
 
 ## AsyncAPI Special Interest Group (SIG) open meeting
 
-The last meeting took place on Tuesday, 1st of September, 8AM UTC. Meeting notes and recording are available [here](https://github.com/asyncapi/asyncapi/issues/430). 
+The last meeting took place on Tuesday, 1st of September, 8AM UTC. Meeting notes and recording are available [here](https://github.com/asyncapi/asyncapi/issues/430).
 
-The next meeting is scheduled for next [Tuesday, 15th of September, 4PM UTC](https://everytimezone.com/s/4347f3c8). 
+The next meeting is scheduled for next [Tuesday, 15th of September, 4PM UTC](https://everytimezone.com/s/4347f3c8).
 
-We work on the agenda for the next meeting [here](https://github.com/asyncapi/asyncapi/issues/435). At the moment, there is nothing in the agenda so you can sneak in your topic easily. 
+We work on the agenda for the next meeting [here](https://github.com/asyncapi/asyncapi/issues/435). At the moment, there is nothing in the agenda so you can sneak in your topic easily.
 
 We host the meeting on [Zoom](https://zoom.us/j/165106914). Do not forget about future meetings and always have up to date invitations in your calendar by adding your email to [this](https://groups.google.com/forum/#!forum/asyncapi-users) mailing list.
```

Comment:
@KhudaDad414 I think you have to update this with the calendar link and Zoom meeting link.